src/redux/news/news.reduser.js

```diff
@@ -0,0 +1,24 @@
+import { NEWS_LOADED, SET_NEWS } from './news.types';
```

Comment:
Please, rename the file from `news.reduser` to `news.reducer`

Reply:
done